pennylane_lightning/src/rework/Util.hpp

```diff
@@ -44,3 +44,9 @@ namespace Pennylane {
     }
 
 }
+
+// Helper similar to std::make_unique from c++14
+template<typename T, typename... Args>
+std::unique_ptr<T> make_unique(Args&&... args) {
+    return std::unique_ptr<T>(new T(std::forward<Args>(args)...));
+}
```

Comment:
Why did we add this? I thought antal removed c++11 support.

Reply:
This was added so that we can keep compiling with c++11. `std::make_unique` was the only feature we would need from c++14 at the moment.

Reply:
:thinking: but aren't we already compiling in c++11 in `master` and things are working?

Reply:
We aren't compiling the `rework/Gates.cpp` source on `master` at the moment when running the tests.